src/resources/translations/en_US.js

```diff
@@ -0,0 +1,102 @@
+export default {
```

Comment:
it's better use static json files instead of javascript files for the translations bundles